### PR TITLE
Added documentation improvements suggested in issue #10 by @Grinnz

### PR DESCRIPTION
**Pull request by jonasbn**
Added suggested documentation improvements to the manual process as outlined by @Grinnz in issue #10 

I created this PR, since it seemed this issue had staled.

### CODE REVIEW

File: lib/Dist/Zilla/Plugin/InstallGuide.pm [hunk 1]
```diff
@@ -63,6 +63,11 @@ Alternatively, if your CPAN shell is set up, you should just be able to do:
 ## Manual installation
 
 {{ $manual_installation }}
+
+The prerequisites of this distribution will also have to be installed manually. The
+prerequisites are listed in one of the files: MYMETA.yml or MYMETA.json generated by
```
Comment (doherty) [inline lib/Dist/Zilla/Plugin/InstallGuide.pm]: Put backticks around the filenames

File: lib/Dist/Zilla/Plugin/InstallGuide.pm [hunk 2]
```diff
@@ -63,6 +63,11 @@ Alternatively, if your CPAN shell is set up, you should just be able to do:
...
+running the manual build proces described above.
```
Comment (doherty) [inline lib/Dist/Zilla/Plugin/InstallGuide.pm]: "process"

**Review by doherty — CHANGES_REQUESTED** [review]

File: lib/Dist/Zilla/Plugin/InstallGuide.pm (hunk 2 above)
Comment (jonasbn) [reply lib/Dist/Zilla/Plugin/InstallGuide.pm]: Spelling corrected

File: lib/Dist/Zilla/Plugin/InstallGuide.pm (hunk 1 above)
Comment (jonasbn) [reply lib/Dist/Zilla/Plugin/InstallGuide.pm]: Hi @doherty

I am not sure what you mean, the endpoint is a plain text file, as for PR #14.

Please enlighten me

Comment (jonasbn) [reply lib/Dist/Zilla/Plugin/InstallGuide.pm]: I have added backticks as requested :-)

**Pull request merged**